recipe/meta.yaml

```
@@ -14,156 +15,227 @@
     sha256: {{ sha256 }}
     patches:
       - support_new_cuda_layout.diff  # [((linux or win64) and (cuda_compiler_version or "").startswith("12")) or (build_platform != target_platform)]
-      - support_nvtx3_win.diff        # [win64]
 
 build:
-  number: 2
-  skip: true  # [cuda_compiler_version not in ("11.8", "12.0")]
-  script:
-    # CuPy default detects CUDA from nvcc, but on Conda-Forge's dockers nvcc lives in a different place...
-    # With conda-forge/nvcc-feedstock#58, CUDA_PATH is set correctly
-    - export NVCC="$(which nvcc)"                                                # [linux]
-    {% if cuda_major >= 12 %}
-    - export CUPY_NUM_BUILD_JOBS=1                                               # [(aarch64 or ppc64le) and (build_platform == target_platform)]
-    - export CUDA_PATH=$PREFIX/targets/{{ target_name }}                         # [linux]
-    {% endif %}
-    - echo "nvcc is $NVCC, CUDA path is $CUDA_PATH"                              # [linux]
-    - for /f "tokens=* usebackq" %%f in (`where nvcc`) do (set "dummy=%%f" && call set "NVCC=%%dummy:\=\\%%")  # [win]
-    # we use this WAR until we figure out how to proceed with conda-forge/nvcc-feedstock#88
-    - set "NVCC=%NVCC% --use-local-env"  # [win64]
-    - echo "nvcc is %NVCC%, CUDA path is %CUDA_PATH%"  # [win]
-    - export CUSPARSELT_PATH="${PREFIX}"                                         # [(linux64 or aarch64 or win) and (cuda_compiler_version or "").startswith("11")]
-    # Workaround __ieee128 error; see https://github.com/LLNL/blt/issues/341
-    - export NVCC="$NVCC -Xcompiler -mno-float128"  # [ppc64le]
+  number: {{ build_num }}
+  skip: true  # [(py < 39) or cuda_compiler_version not in ("11.8", "12.0")]
 
-    - {{ PYTHON }} -m pip install . --no-deps -vv
-    - if errorlevel 1 exit 1  # [win]
+outputs:
 
-    # copy activate/deactivate scripts
-    - mkdir -p "${PREFIX}/etc/conda/activate.d"                                               # [linux]
-    - cp "${RECIPE_DIR}/activate.sh" "${PREFIX}/etc/conda/activate.d/cupy_activate.sh"        # [linux]
-    - mkdir -p "${PREFIX}/etc/conda/deactivate.d"                                             # [linux]
-    - cp "${RECIPE_DIR}/deactivate.sh" "${PREFIX}/etc/conda/deactivate.d/cupy_deactivate.sh"  # [linux]
-    - if not exist %PREFIX%\etc\conda\activate.d mkdir %PREFIX%\etc\conda\activate.d          # [win]
-    - copy %RECIPE_DIR%\activate.bat %PREFIX%\etc\conda\activate.d\cupy_activate.bat          # [win]
-    - if not exist %PREFIX%\etc\conda\deactivate.d mkdir %PREFIX%\etc\conda\deactivate.d      # [win]
-    - copy %RECIPE_DIR%\deactivate.bat %PREFIX%\etc\conda\deactivate.d\cupy_deactivate.bat    # [win]
+  - name: cupy
+    version: {{ version }}
+    build:
+      number: {{ build_num }}
+      skip: true  # [(py < 39) or cuda_compiler_version not in ("11.8", "12.0")]
+    requirements:
+      build:
+      host:
+      run:
```

Think we need to add `python` as a dependency here

```suggestion
      host:
        - python
      run:
        - python
```

---

Don't we get it from `cupy-core`?